pkg/fuzzy/google_translate.go

```diff
@@ -271,7 +271,7 @@ func patternToString(pattern []ast.Pattern) string {
 		switch v := pattern[i].(type) {
 		case ast.TextPattern:
 			text += string(v)
-		case ast.PlaceholderPattern:
+		case ast.Pattern:
 			text += fmt.Sprintf("{$%d}", i)
```

Comment:
Same here
```suggestion
		case ast.LiteralExpression, ast.AnnotationExpression, ast.VariableExpression, ast.Markup: // or default
			text += fmt.Sprintf("{$%d}", i)
		}
```

Reply:
Update godoc, there is no more `PlaceholderPattern`.
Patterns are:
- Text
- All Expressions (Literal, Variable, Annotation)
- Markup